app/views/scc_accounts/index.html.erb

```diff
@@ -25,7 +25,7 @@
             display_link_if_authorized(_("Sync"), hash_for_sync_scc_account_path(:id => scc_account).merge(:auth_object => scc_account, :authorizer => authorizer),
                                        :method => :put),
             display_delete_if_authorized(hash_for_scc_account_path(:id => scc_account).merge(:auth_object => scc_account, :authorizer => authorizer, :permission => 'delete_scc_accounts'),
-                                         :data => { :confirm => _("Delete %s?") % scc_account.to_s })
+                                         :data => { :confirm => _("WARNING: If you want to switch SCC accounts and retain the synchronized content, DO NOT delete your old SCC account, even if it is expired. Please change the login and password of your SCC account, instead. If you delete your old SCC account, you CANNOT reuse existing repositories, products, content views, and composite content views. Really delete SCC account %s?") % scc_account.to_s })
```

Comment:
Maybe add some newlines, at least between the Warning description and the `Really delete SCC account...`.

Reply:
Good point